extension/manifest.json

```diff
@@ -25,6 +25,7 @@
   "background": {
     "persistent": true,
     "scripts": [
+      "sentry.js",
```

Comment:
can we add this file to `model\static\errorReporting`?

Reply:
done